src/delagent/mod_deps

```diff
@@ -36,10 +36,10 @@ if [[ $? -ne 0 ]]; then
   OPTS='--help'
 fi
 
-eval set -- "$OPTS"
```

Comment:
The change `$OPTS` -> `${OPTS}` is unnecessary for the most variables

Reply:
Done.